### ISSUE TITLE
Can no longer connect to nodes in a pool from Explorer

### ISSUE THREAD

**Issue by msplants** [Azure/BatchExplorer#2129]
When I try to Remote Desktop to a node from Batch Explorer, no Remote Desktop login window is generated.  The Connect button just spins.  I am able to Remote Desktop from Batch Explorer if I use use Version 2.3.1-stable.378.

**To Reproduce**
1. Go to Pools
2. Click on a Pool
3. Right click a node and click Connect
4. Click on "Connect" button in "connect to node" form
5. Connect button has spinning circle but no Remote Desktop window is ever generated

**Expected behavior**
I expected to see a Remote Desktop login prompt.

**Logs(Click profile > View Logs > app.log, client.log)**
app.log

```
{"0":"Duration.fromObject: argument expected to be an object, got object","level":"error","message":"[BL] Uncaught exception:","timestamp":"2019-11-22T19:33:38.718Z"}
{"0":"Duration.fromObject: argument expected to be an object, got object","level":"error","message":"[BL] Uncaught exception:","timestamp":"2019-11-22T19:35:44.738Z"}
{"0":"Duration.fromObject: argument expected to be an object, got object","level":"error","message":"[BL] Uncaught exception:","timestamp":"2019-11-22T19:38:01.134Z"}
{"0":"Duration.fromObject: argument expected to be an object, got object","level":"error","message":"[BL] Uncaught exception:","timestamp":"2019-11-22T19:38:35.838Z"}
{"0":"409 - OK - ","level":"error","message":"[BL] Uncaught exception:","timestamp":"2019-11-22T19:48:09.197Z"}
{"0":"409 - OK - ","level":"error","message":"[BL] Uncaught exception:","timestamp":"2019-11-22T19:48:09.206Z"}
{"0":{"details":[],"message":"","original":{"error":{"error":{"code":"AccountNotEnabledForAutoStorage","message":"The Batch account does not have a linked auto-storage account.\nRequestId:84508f39-3d4a-4061-bb3d-7837e38eb250\nTime:2019-11-22T19:48:08.9093895Z","target":"BatchAccount"}},"headers":{"headers":{},"lazyInit":null,"lazyUpdate":null,"normalizedNames":{}},"message":"Http failure response for https://management.azure.com/subscriptions/79a0da7f-7778-4b81-99a3-293b14111c2a/resourcegroups/default-azurebatch-westus/providers/microsoft.batch/batchaccounts/kirkolenslego/applications?$top=10&api-version=2019-08-01: 409 OK","name":"HttpErrorResponse","ok":false,"status":409,"statusText":"OK","url":"https://management.azure.com/subscriptions/79a0da7f-7778-4b81-99a3-293b14111c2a/resourcegroups/default-azurebatch-westus/providers/microsoft.batch/batchaccounts/kirkolenslego/applications?$top=10&api-version=2019-08-01"},"requestId":"84508f39-3d4a-4061-bb3d-7837e38eb250","status":409,"statusText":"OK","timestamp":"2019-11-22T19:48:07.000Z"},"level":"error","message":"Error loading data in ListView for BatchApplication","timestamp":"2019-11-22T19:48:09.250Z"}
{"0":"Duration.fromObject: argument expected to be an object, got object","level":"error","message":"[BL] Uncaught exception:","timestamp":"2019-11-22T19:48:49.782Z"}
```

**Version (please complete the following information):**
 - OS: Windows
 - Version: 18363.476
- Batch Explorer Version: 2.4.1-stable.392

**Comment by bgklein** [comment]
I believe this has been fixed by #2120 , so switching to the insider build should resolve the issue until we do our next release (Profile -> settings -> Update Channel [change to insider])

**Comment by msplants** [comment]
That fixes it.  Thanks!
